### PR TITLE
added java guide without build errors

### PR DESCRIPTION
**Pull request by williamong68**

### CODE REVIEW

File: docs/connecting-to-tembo/Java.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+It is important to use `Java 8` or greater and `PostgreSQL 4.2 JDBC Driver`
```
Comment (sjmiller609) [inline docs/connecting-to-tembo/Java.md:1]: should this say something like ...
```suggestion
It is important to use Java version 8 or greater and PostgreSQL JDBC Driver version 4.2 or greater.
```

File: docs/connecting-to-tembo/Java.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+We recommend downloading the latest Java Development Kit  [JDK 21](https://www.oracle.com/java/technologies/downloads/#java8-mac)
+Also, you must download the [PostgreSQL 4.2 JDBC Driver](https://jdbc.postgresql.org/download/)
```
Comment (sjmiller609) [inline docs/connecting-to-tembo/Java.md:3]: Is this specific version required, or is it this version, or greater?

File: docs/connecting-to-tembo/Java.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```java title="Example.java"
+import java.sql.*;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class Example
+{
+   public static void main(String[] args)
+   {
+      try
```
Comment (sjmiller609) [inline docs/connecting-to-tembo/Java.md:14]: Most of the guides have a try / catch block, but is that really necessary? why not just skip the try / catch

**Review by sjmiller609 — APPROVED** [review]
looks good! a few tips

**Review by ericankenman — APPROVED** [review]

**Pull request merged**